README.md

```diff
@@ -1,37 +1,46 @@
-# Travel and Help
-[![Build Status](https://travis-ci.org/travel-and-help/start-kit.svg?branch=master)](https://travis-ci.org/travel-and-help/start-kit)
-[![Coverage](https://codecov.io/github/travel-and-help/start-kit/coverage.svg?branch=master)](https://codecov.io/github/travel-and-help/start-kit?branch=master)
-
-## Links
-
-#### CI [Travis-CI] (https://travis-ci.org/travel-and-help/start-kit)
-
-#### E2E [Sauce Labs] (https://saucelabs.com/beta/dashboard/tests)
-
-#### LIVE [Heroku] (https://start-kit.herokuapp.com/)
-
-#### Style Gude [Airbnb React/JSX Style Guide] (https://github.com/airbnb/javascript/tree/master/react#naming)
-
-## Main npm scripts
-
-* ``` npm run watch ``` build app, stand up local mongo db, start server and watch for changes on bacend and frontend (best for development). Notice: front end will be served from memory by webpack dev server and be available on __PORT + 1__
-* ``` npm start ``` build project (target depends on __NODE_ENV__) and serve target (best for deployment)
-
-* ``` npm run lint ``` lint all sources (see [JS style guide](https://github.com/airbnb/javascript))
-* ``` npm run lint:front ``` lint frontend sources
-* ``` npm run lint:back ``` lint backend sources
-
-* ``` npm test ``` run all tests
-* ``` npm run test:front ``` run frontend tests
-* ``` npm run test:back ``` run backend tests
-
-* ``` npm run coverage ``` generates html coverage report
-
-* ``` npm run test-ci ``` lint code, run tests, generates coverage report and uploads it to codecov
+# Travel and Help: React Team [![Build Status](https://travis-ci.org/travel-and-help/start-kit.svg?branch=master)](https://travis-ci.org/travel-and-help/start-kit) [![Coverage](https://codecov.io/github/travel-and-help/start-kit/coverage.svg?branch=master)](https://codecov.io/github/travel-and-help/start-kit?branch=master)
+
+### System requirements:
+ - Node -v 5.10.1
+ - NPM -n 3.8.3
+ - Cordova -v 6.1.1
+ - Java -version 1.8.0_45
+ 
+### Continuous integration [Travis-CI] (https://travis-ci.org/travel-and-help/start-kit)
+
+### Integration tests [Sauce Labs] (https://saucelabs.com/beta/dashboard/tests)
+
+### UAT Environment [Heroku] (https://start-kit.herokuapp.com/)
+
+### JavaScript Style Guide [Airbnb] (https://github.com/airbnb/javascript)
+
+### Start development:
+```sh
+$ npm start
+```
+
+### Run unit tests:
+```sh
+$ npm test
+```
+
+### Build application:
+```sh
+$ npm run build
+```
+
+### Emulate with cordova:
+```sh
+$ NODE_ENV=cordova
```

Comment:
It is hard to remember this command sequence, it will be cool if one npm script will take care of it